### DISCUSSION TITLE
Cannot the find file \stable-diffusion-webui-master\src\

### DISCUSSION THREAD

**Discussion by easonoob** [hlky/stable-diffusion-webui#1117] [Q&A] [answered]
I cannot find the file` \stable-diffusion-webui-master\src\` to add the GFPGAN model, do I need to create one?

**Comment by codedealer** [comment, marked as answer]
create the environment first as outlined in the guide, it will create `src` for you automatically, then you can return and drop in models there

**Reply by easonoob** [reply]
ok, thanks!
